src/options/proof_options.toml

```diff
@@ -33,6 +33,14 @@ name   = "Proof"
   default    = "false"
   help       = "Print conclusion of proof steps when printing AST"
 
+[[option]]
```

Comment:
Rather than a new option, why not use the information in --dag-thresh?

Reply:
These are orthogonal options.

One can choose (1) whether to globally dagify, (2) if global dagification is off, then we print terms based on `dag-thresh`.